addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlGenerator.java

```diff
@@ -41,6 +41,8 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
```

Comment:
Maybe how it's used here is not a problem but it would be preferable not to use these classes, they have quirks (i.e. `JsonUtil`).